src/components/CheckBox/CheckBox.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+const CheckBox = ({ id, text, spaceBetween, fontSize, margin }) => {
+  return (
+    <Wrapper>
+      <CheckBoxContainer margin={margin}>
+        <StyledInput type="checkbox" id={id} spaceBetween={spaceBetween} />
+        <StyledLabel htmlFor={id} fontSize={fontSize}>
+          {text || '모름'}
+        </StyledLabel>
+      </CheckBoxContainer>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div``;
+
+const CheckBoxContainer = styled.span`
+  display: flex;
```

Comment:
```suggestion
const CheckBoxContainer = styled.div`
  display: flex;
```

flex는 외부 디스플레이가 block처럼 동작하는 것으로 알고 있습니다. span을 써도 block으로 동작하기 때문에 div여도 될 것 같습니다.